basiclti-impl/src/java/org/sakaiproject/lti/impl/BaseLTIService.java

```diff
@@ -650,6 +650,11 @@ public Object insertToolContent(String id, String toolId, Properties reqProps, S
 		return insertToolContentDao(id, toolId, reqProps, siteId, isAdmin(siteId), isMaintain(siteId));
 	}
 	
+	public Object insertTIIToolContent(String id, String toolId, Properties reqProps)
```

Comment:
Just discussed on the call and the plan is to not change the API and to use `insertToolContent(id, toolId, reqPropes, “!admin”)`.